db/migrate/20160914161314_create_fast_versioning_fast_versions.rb

```diff
@@ -1,4 +1,4 @@
-class CreateFastVersioningFastVersions < ActiveRecord::Migration
+class CreateFastVersioningFastVersions < ActiveRecord::Migration[4.2]
```

Comment:
I think that syntax will actually fail on rails 4, we probably need a `Rails::VERSION::MAJOR` conditional for this like ie. [here](https://github.com/plataformatec/devise/blob/master/lib/generators/active_record/devise_generator.rb#L94)